docs/jwks.md

```diff
@@ -0,0 +1,63 @@
+# JWKS
```

Comment:
@carl-codeorg I added additional documentation on the process for rotating, let me know if this is better or if there is still room for additional clarification.

Reply:
Yeah it's much clearer now! Thanks